libs/components/forms/src/index.ts

```diff
@@ -48,6 +48,7 @@ export { SkyFormErrorsComponent as λ21 } from './lib/modules/form-error/form-er
 export { SkyFormErrorComponent as λ22 } from './lib/modules/form-error/form-error.component';
 export { SkyInputBoxControlDirective as λ20 } from './lib/modules/input-box/input-box-control.directive';
 export { SkyInputBoxComponent as λ10 } from './lib/modules/input-box/input-box.component';
+export { SkyInputBoxHintTextPipe as λ24 } from './lib/modules/input-box/input-box-hint-text.pipe';
```

Comment:
Does this need to be publicly exported?

Reply:
Good call - I was thinking it did - but since it is only used internally - I don't believe it does need to be exported.